src/quiz/Description.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState, useEffect } from "react";
+import { useHistory } from "react-router-dom";
+import axios from "axios";
+import "./css/style.css";
+import "./css/Description.css";
+import Timer from "../common/Timer";
+
+export default function Description() {
+  const [data, setData] = useState({
+    playerId: [
+      {
+        playerDescription: "百獣の王は静かに微笑みを湛えている",
+        aiDescription: "草原でライオンが座っています",
+        ngWord: ["草原", "ライオン"],
+      },
+      {
+        playerDescription: "幾重の鳥居が私たちを待っている",
+        aiDescription: "草に囲まれた赤い建物に光が当たっています",
+        ngWord: ["草", "赤い", "光"],
+      },
+      {
+        playerDescription: "プレイヤー説明文",
+        aiDescription: "AI説明文",
+        ngWord: ["NG1", "NG2", "NG3"],
+      },
+      {
+        playerDescription: "プレイヤー説明文",
+        aiDescription: "AI説明文",
+        ngWord: ["NG1", "NG2", "NG3"],
+      },
+    ],
+  });
+  const [ngWord, setNgWord] = useState("");
+  const [attentionMessage, setAttentionMessage] = useState("");
+  const [myDescription, setMyDescription] = useState("");
+  const history = useHistory();
+  const [time, setTime] = useState(30);
+
+  useEffect(() => {
+    // axios.get("url")
+    //     .then(res => {
+    //         setData(res.data);
+    //     })
```

Comment:
```suggestion
    // TODO: APIとの通信
```

WebStormだとコメントにTodoかけるのでこうしましょう